Gemfile

```diff
@@ -11,5 +11,4 @@ gem 'should_not', '~> 1.1'
 
 gem 'rake', '~> 10.4.2'
 
-# The default latest json 1.8.3 fails on some macs
-gem 'json', '= 1.8.1'
+gem 'json', '= 1.8.6'
```

Comment:
Should we use 2.1.0 of this as that's what ships with Ruby 2.5.1

Reply:
As this is used only in the tests there is not much risk to go for 2.1.0